docs/source/open-software-summer-school/2026/large-data.md

```diff
@@ -0,0 +1,66 @@
+(track-large-array-data-2026)=
+# Track: Large Array Data
+
+In many areas of research, large array datasets have become important and wide-spread. Examples include whole-organ images (e.g. light-sheet, HiP-CT), Neuropixels data, and long behavioural videos — to name just a few. Processing, analyzing and visualising these datasets is made significantly more complex and slow because they do not fit into a typical computer's memory and take up a lot of disk space.
+
+The use of compression algorithms and "chunked" file formats (which favour parallel processing) enable us to deal with these challenges effectively, but doing this well requires careful consideration of the trade-offs between read and write speeds, disk space used, and portability.
```

Comment:
```suggestion
The use of compression algorithms and modern file formats which favour parallel processing enable us to deal with these challenges effectively, but doing this well requires careful consideration of the trade-offs between read and write speeds, disk space used, and portability.
```
A suggestion to avoid "chunked"